pkg/infrastructure/clusterapi/clusterapi.go

```diff
@@ -51,6 +51,9 @@ const (
 	ignitionStage            = "Bootstrap Ignition Provisioning"
 	machineStage             = "Machine Provisioning"
 	postProvisionStage       = "Infrastructure Post-provisioning"
+
+	// CAPIGeneratedAssetDir is the directory where the manifests generated by CAPI are stored.
+	CAPIGeneratedAssetDir = ".capi_output"
```

Comment:
```
// CAPIGeneratedAssetDir is the directory where the manifests generated by CAPI are stored.
	CAPIGeneratedAssetDir = ".capi_output"
```

Can we  use `Artifact` instead of `Asset`? So it's something like `ClusterAPIArtifactDir` or `CAPIArtifactDir`? I don't think we should limit the contents to assets, if that makes sense.

Reply:
Done.